ci/build_python.sh

```diff
@@ -3,24 +3,26 @@
 
 set -euo pipefail
 
-rapids-configure-conda-channels
```

Comment:
🎉 thank you! Getting closer on https://github.com/rapidsai/gha-tools/issues/145 😁